controller/controller_test.go

```diff
@@ -626,12 +626,12 @@ func (s *TestSuite) TestIsSameGuaranteedCPURequirement(c *C) {
 	c.Assert(err, IsNil)
 	c.Assert(IsSameGuaranteedCPURequirement(a, b), Equals, true)
 
-	b.Requests[corev1.ResourceCPU], err = resource.ParseQuantity("250m")
+	b.Requests[corev1.ResourceCPU], _ = resource.ParseQuantity("250m")
```

Comment:
Suggest to `c.Assert(err, IsNil)` instead of overlooking it. The same applies to the code below.

Reply:
Arguably, we should be able to assume that the `resource.ParseQuantity` can correctly parse the values given. It's not the point of our unit tests to make sure of that.
However I see the point that we should also make sure we pass parseable values to `resource.ParseQuantity`.